src/hooks/useMarketStats.tsx

```diff
@@ -273,11 +273,11 @@ async function getKwentaStatsFetcher(
   const positionValue = wei(margin).mul(leverage).div(sUSDRate).toBN()
   const oneHourFunding = oneHourlyFundingRate.gt(ZERO_BIG_NUM)
     ? positionSide === 'long'
-      ? wei(marketData.assetPrice).mul(oneHourlyFundingRate).neg().toBN()
-      : wei(marketData.assetPrice).mul(oneHourlyFundingRate).toBN() // positive && short position
+      ? susdSize.mul(oneHourlyFundingRate).neg().toBN()
+      : susdSize.mul(oneHourlyFundingRate).toBN() // positive && short position
     : positionSide === 'short'
-    ? wei(marketData.assetPrice).mul(oneHourlyFundingRate).toBN()
-    : wei(marketData.assetPrice).mul(oneHourlyFundingRate).abs().toBN() // negative && long position
```

Comment:
The funding rate unit is in the native asset as well. I believe you still want the price, something like: `wei(marketData.assetPrice).mul(sizeDelta).mul(oneHourlyFundingRate)`